### DISCUSSION TITLE
Really Important question about my account.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#72254] [GitHub Education]
### Hi everyone,

I have student pack claimed in my account, but for some reason I want to switch to a new github account.
If I downgrade the plan or delete the github account with the pro version, will i be able to claim the student pack again on my new account?
please answer

**Comment by stolenfallen1** [comment]
Hi, 

If you delete your GitHub account with pro version it will obviously lose its "pro version" benefits together with the account. I think you can try claiming the pack on a new account, but you need to do over the verification and eligibility of your new account.

**Reply by anandpanda** [reply]
Ok then, I am deleting the account.
Thanks.
